<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebWindow</class>
 <widget class="QMainWindow" name="WebWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>411</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainContent">
   <widget class="MouseCaptureWidget" name="ClickArea" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>651</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QWebView" name="WebView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>34</y>
      <width>651</width>
      <height>235</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QFrame" name="NavigationBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>466</width>
      <height>34</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/Resources/BackIcon.png</normaloff>:/icons/Resources/BackIcon.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="forwardButton">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>1</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/Resources/ForwardIcon.png</normaloff>:/icons/Resources/ForwardIcon.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>401</x>
       <y>1</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/Resources/StopButton.png</normaloff>:/icons/Resources/StopButton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reloadButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>1</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/Resources/RefreshButton.png</normaloff>:/icons/Resources/RefreshButton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="addressBar">
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>3</y>
       <width>302</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>http://</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clippingModeButton">
     <property name="geometry">
      <rect>
       <x>432</x>
       <y>1</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/Resources/ClipIcon.png</normaloff>:/icons/Resources/ClipIcon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewWindow"/>
    <addaction name="actionGo"/>
    <addaction name="actionRestoreWindow"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionForward"/>
    <addaction name="actionBack"/>
    <addaction name="actionStop"/>
    <addaction name="actionReload"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNavigation"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewWindow">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionRestoreWindow">
   <property name="text">
    <string>Restore Window</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionGo">
   <property name="text">
    <string>Go</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>MouseCaptureWidget</class>
   <extends>QWidget</extends>
   <header>MouseCaptureWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>WebView</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
